crates/solvers/src/util/bytes.rs

```diff
@@ -0,0 +1,30 @@
+/// A thin wrapper around a collection of bytes. Provides hex debug formatting.
```

Comment:
We have by now 3 different ways how to handle this.
1. `Bytes` wrapper
2. [`hex.rs`](https://github.com/cowprotocol/services/blob/main/crates/solvers/src/util/fmt/hex.rs)
3. using `Derivative` + `shared::debug_bytes`

I think we should pick one and use it everywhere but IMO using the wrapper type should not be the way. This forces you to go through another layer of indirection even when you don't use the `Debug` implementation which doesn't seem great.

Reply:
Would you like to create a cleanup ticket for this to make it consistent across repo? might be a good first issue.

Reply:
Tracked https://github.com/cowprotocol/services/issues/2030